lib/ansible/modules/copy.py

```diff
@@ -121,6 +121,7 @@
 - decrypt
 - files
 - validate
+- action_common_attributes
```

Comment:
So, anyone that adds `doc['attributes']` also needs to add this fragment?

Reply:
Or just create their own, this fragment shows the 'standard ones' that the core engine acknowledges

Another option is to autoadd this fragment to all actions.

Reply:
I like adding it by default so people don't shoot themselves in the foot with the standard attributes.

I like that people can add their own fragment with extra attributes. Maybe that will be useful for some collections.

Reply:
I'm 50/50 on that since I also wanted this to be 'meaningful' for core/engine and allow controller to make decisions based on the docs w/o having to execute the module.

Still WIP, see irc core meeting log, this PR is mostly 'pasta on wall' and will take a while before its ready to merge for next ansible version.